code/datums/uplink_item.dm

```diff
@@ -1181,6 +1181,9 @@ var/list/uplink_items = list()
 	if(item)
 		if(findtext(item, /obj/item/organ/internal/cyberimp))
 			return new /obj/item/weapon/storage/box/cyber_implants(loc, item)
+			U.uses -= max(cost, 0)
+			U.used_TC += cost
+			feedback_add_details("traitor_uplink_items_bought", name) //this one and the line before copypasted because snowflaek code
```

Comment:
I have a feeling you guys shouldn't let me code when I'm tired. 

Yeah, I didn't actually test this one. 

My bad. :v